validation/tdd_latest.xsd

```diff
@@ -209,12 +209,16 @@ as passed-in parameters.
     <xs:restriction base="xs:string">
       <xs:enumeration value="Case"/>
       <xs:enumeration value="BalancedIIF"/>
+      <xs:enumeration value="BalancedIIF_"/>
+      <xs:enumeration value="BalancedIF"/>
```

Comment:
```suggestion
      <xs:enumeration value="BalancedIIF_"/> <!-- added 2021.4 -->
      <xs:enumeration value="BalancedIF"/> <!-- added 2021.4 -->
```
I put this comment on the other branch as well.  If for some reason comments in these XSDs aren't valid then ignore this feedback.  The packager tests should tell us either way.

Reply:
tested those comments with the packager, it works.